### PR TITLE
Make RepoActionView.vue support `##[group]`

### PR DESCRIPTION
**Pull request by wxiaoguang**

### CODE REVIEW

**Comment by silverwind** [comment]
Can this be written in a testable fashion? E.g. actions output in and assert presence of certain HTML elements in rendered output.

**Comment by wxiaoguang** [comment]
> Can this be written in a testable fashion?

By the way, all of "parseLineCommand", "beginLogGroup", "endLogGroup" and "appendLogs" are testable, as long as export them. Intentionally I decoupled them from the Vue SFC. While I guess we do not need to do that test in this PR.

File: web_src/js/components/RepoActionView.vue [hunk 1]
```diff
@@ -393,7 +413,7 @@ export function initRepositoryActionView() {
         <button class="ui basic small compact button red" @click="cancelRun()" v-else-if="run.canCancel">
           {{ locale.cancel }}
         </button>
-        <button class="ui basic small compact button tw-mr-0 tw-whitespace-nowrap link-action" :data-url="`${run.link}/rerun`" v-else-if="run.canRerun">
+        <button class="ui basic small compact button link-action" :data-url="`${run.link}/rerun`" v-else-if="run.canRerun">
```
Comment (silverwind) [inline web_src/js/components/RepoActionView.vue:416]: - `tw-mr-0` should eliminate a 4px margin-right caused by button CSS, can we restore it? I was meaning to remove this margin from CSS but have not gotten around to it yet.
- `tw-whitespace-nowrap` should prevent the button text from ever wrapping, like on small viewport. Did you check?

Comment (silverwind) [reply web_src/js/components/RepoActionView.vue:416]: Ah I see you put replacement CSS below. Is margin between buttons still present?

Comment (wxiaoguang) [reply web_src/js/components/RepoActionView.vue:416]: These buttons are all in `if-else`

The `tw-mr-0` you added was only on the last `else` button.

**Review by silverwind — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Pull request merged**